I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script/main.rpy", line 22, in script
    return
Exception: Could not find return label (u'game/script/main.rpy', 1628607418, 17).

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script/main.rpy", line 22, in script
    return
  File "/Applications/renpy-7.4.4-sdk/renpy/ast.py", line 1521, in execute
    next_node(renpy.game.context().lookup_return(pop=True))
  File "/Applications/renpy-7.4.4-sdk/renpy/execution.py", line 710, in lookup_return
    raise Exception("Could not find return label {!r}.".format(self.return_stack[-1]))
Exception: Could not find return label (u'game/script/main.rpy', 1628607418, 17).

Darwin-20.6.0-x86_64-i386-64bit
Ren'Py 7.4.4.1439
WonderWhy 1.0
Tue Aug 10 22:59:55 2021
